### PR TITLE
chore: bump govuk-frontend version and update svg with tudor crown

### PR DESCRIPTION
**Pull request by jrabbott**
### Context
Update to use Tudor crown.

### Change proposed in this pull request
Service using GOV.UK branding, must update between 19 February and 1 March 2024 to use the new logo.

### Guidance to review 
https://github.com/alphagov/govuk-frontend/releases/tag/v5.1.0

### Checklist
- [x] ~~Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)~~
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -38,17 +38,16 @@
...
-                        </path>
+                        width="148"
+                        aria-label="GOV.UK">
+                        <title>GOV.UK</title>
```
Comment (WolfyUK) [inline web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:49]: Duplicate GOV.UK elements. It looks like it's the one lower down that is not required.
image

**Comment by WolfyUK** [comment]
Should the favicons be included here, or is that a separate PR?

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (jrabbott) [reply web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:49]: Interesting didn't see that locally when I tested. Will double check.

**Comment by jrabbott** [comment]
Fav icons should be included with the bump of the package version.

**Comment by WolfyUK** [comment]
> > Fav icons should be included with the bump of the package version.
> 
> I'm still getting the old ones coming through, even after a hard-refresh. I'll try again

`favicon.ico` seems correct, but the `.png` and `.ico` variants are still the old version

**Comment by WolfyUK** [comment]
> > > Fav icons should be included with the bump of the package version.
> > 
> > 
> > I'm still getting the old ones coming through, even after a hard-refresh. I'll try again
> 
> `favicon.ico` seems incorrect, but the `.png` and `.ico` variants are using the correct version

Oh, it looks like there are two versions of the `favicon.ico`. The one in `wwwroot` appears different from the one in `wwwroot/assets/images`

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (jrabbott) [reply web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:49]: Sorted - pushed changes

**Comment by jrabbott** [comment]
Re: images & icons - cleared down wwwroot locally & reran gulp task, all the assets seem to be correct.

**Review by WolfyUK — APPROVED** [review]

**Pull request merged**